src/prefect/cli/agent.py

```diff
@@ -1,11 +1,18 @@
 import click
+import warnings
 
 from prefect import config
 from prefect.utilities.configuration import set_temporary_config
 from prefect.utilities.serialization import from_qualified_name
 from prefect.utilities.cli import add_options
 
 COMMON_START_OPTIONS = [
+    click.option(
+        "--api-key",
+        "-k",
+        required=False,
+        help="A Prefect Cloud API Key with RUNNER scope. (DEPRECATED)",
```

Comment:
This help string is mixing the two concepts together in an inaccurate way; an API key should be provided here, there is no concept of "RUNNER" scoping, and API keys are not deprecated as this string suggests